homeassistant/components/habitica/coordinator.py

```diff
@@ -56,7 +56,13 @@ async def _async_update_data(self) -> HabiticaData:
         try:
             user_response = await self.api.user.get()
             tasks_response = await self.api.tasks.user.get()
-            tasks_response.extend(await self.api.tasks.user.get(type="completedTodos"))
+            tasks_response.extend(
+                [
+                    {**task, "id": task["_id"]}
```

Comment:
Should we make sure "id" also works so we don't break again when it gets fixed?

Reply:
Usually the response has both fields, `_id` and `id`. Internally the Habitica API simply copies the value from `_id` to `id` and here I am doing the same, simply copying `_id` to `id`. This would have no effect when it is fixed but I changed it to only copy the value in case `id` is not available